FormHW4.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const { format } = require("path");
+const app = express();
+const path = require("path");
+const port = 3000;
+const bodyparse = require("body-parser");
+
+app.use(bodyparse.urlencoded({ extended: false }));
+
+app.get("/form", (req, res) => {
+  res.sendFile(__dirname + "/form.html");
+});
+
+app.post("/formsubmit", (req, res) => {
+  res.send(`Name : ${req.body.Name} <br> 
```

Comment:
This is easy to follow and straightforward implementation. I would suggest setting a status code to the client